app/src/main/java/vn/mbm/phimp/me/PhotoSelect.java

```diff
@@ -281,7 +281,7 @@ protected String doInBackground(Cursor... cursor) {
 	 				boolean check = PhimpMe.cache.check(path);
 	 				if(!check){	 					
 	 					@SuppressWarnings("deprecation")
```

Comment:
Since you removed the depreciated code can you remove the supress tag as well as we don't need to supress the depreciation as there w9uld be no depreciation!